config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  # Routes for all tasks
+  get '/tasks', to: 'tasks#index', as: 'tasks'
+  get 'tasks/new', to: 'tasks#new', as: 'new_task'
+  post '/tasks', to: 'tasks#create'
+
+  # Routes for using a specific task page
+  get '/tasks/:id', to: 'tasks#show', as: 'task'
+  get '/tasks/:id/edit', to: 'tasks#edit', as: 'edit_task'
+  patch '/tasks/:id', to: 'tasks#update'
+  get '/tasks/:id/confirm_delete', to: 'tasks#confirm', as: 'confirm_task'
+  delete '/tasks/:id', to: 'tasks#delete', as: 'delete_task'
```

Comment:
The action is referred to as `delete` here but it is named `destroy` in the controller. It is important that the names line up for Rails to be about to route the request correctly.
Convention would be to name it `delete` but as long as they match, it will work.